model/fv_regional_bc.F90

```diff
@@ -248,7 +248,7 @@ module fv_regional_mod
 
       integer :: a_step, p_step, k_step, n_step
 !
-      character(len=80) :: data_source
+      integer :: src_grp   ! source group
```

Comment:
How about naming this variable `source_group` or even `data_source_group` instead `src_grp`? Then you do not need trailing comment to explain what it is.

Reply:
Done!